demos/browser/package.json

```diff
@@ -7,7 +7,7 @@
     "build:fast": "tsc && webpack --config ./webpack.config.js",
     "build": "npm run deps && npm install && npm run build:fast",
     "start:fast": "npm run build:fast && node serve.js",
-    "start:hot": "tsc && webpack-dev-server --config ./webpack.config.js",
```

Comment:
Should we also update the `package.json` to remove the `webpack-dev-server` dependency if we are removing it from here?

Reply:
I didn't remove it because I didn't have the time to verify whether it was still needed. Happy to tackle that in a follow-up.

Reply:
Sounds good!